doc/source/whatsnew/v2.1.0.rst

```diff
@@ -172,6 +172,7 @@ and ``na_action="ignore"`` did not work correctly for any :class:`.ExtensionArra
 *New behavior*:
 
 .. ipython:: python
+   :okwarning:
```

Comment:
There is a running example using `na_action`. I can convert it to not running if preferred.

Reply:
I would prefer if we just convert this to a `.. code-block:: python`